assignment-2/power.md

```diff
@@ -1,3 +1,127 @@
 # Power
 
-// TODO
\ No newline at end of file
+##Healthcare.gov out of control
+
+Assignment failed case: 
+>Analyse for the failed case to what extent people get the right amount of power. Take into account: to what extent control is adequate, how power is balanced and how the context of a learning environment is of impact. 
+
+###Context
+On 23 March 2010, president Obama signed the Patient Protection and Affordable Care Act, better known as Obamacare. One of the consequences of this law was that everyone in the USA would require access to a health insurance marketplace. Therefore the federal government started the HealtCare.gov project. The implementation of the law depended crucially on this website. It was therefore imperative for Democratic party to make this project a success. On the other hand the Republican party opposed and still opposes the Patient Protection and Affordable Care Act (Obamacare). 
```

Comment:
"It was therefore imperative for Democratic party" missing 'the' before Democartic party.